.travis.yml

```diff
@@ -46,6 +46,9 @@ after_script:
     npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
     git push --tags git@github.com:$TRAVIS_REPO_SLUG.git &&
     npm publish --tag $NPM_TAG
+  - >
+    [ "${TRAVIS_BRANCH}" = "master" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] &&
+    yarn run publish
```

Comment:
actually i just realized that `publish` probably isn't a good name either because it's ambiguous with a `npm publish` :/ sorry. `publish-storybook`?